src/app/exercise/runner/runner.component.ts

```diff
@@ -235,15 +235,50 @@ function injectIframe(element: any, config: IframeConfig, runner: RunnerComponen
                 font-weight: 300;
                 background-color: #ffe;
               }
+              .mainerror {
+                color: black; 
+                font-size: larger;
+              }
+              .othererrors {
```

Comment:
What is the difference between main errors and other errors? They look the same?